app/views/faq_groups/basics.md

```diff
@@ -52,25 +50,26 @@ A key ingredient to success on Assembly is community support. There are many way
 
 You do. However, in order for Assembly to monetize the product on behalf of you and the community, we need some non-exclusive rights that are outlined in our [terms of service](https://assembly.com/terms).
 
+
 # What does transparency mean to Assembly?
 
 This is really important to us. When a product is built on Assembly, we believe an open, transparent process is key. That means anyone can see progress, discussions, ownership and even revenue.
 
+Obviously we couldn't make the claim that building products in public is the best approach if we weren’t doing it ourselves, so we're entirely open source. Check out [assembly.com/meta](https://assembly.com/meta) to see our development history and discussions, and even get real-time updates as we ship new features.
```

Comment:
It’d be nice to change that link to “Assembly Meta” as opposed to the actual URL